src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryDeploymentProperties.java

```diff
@@ -37,6 +39,8 @@
 
 	public static final String SERVICES_PROPERTY_KEY = CLOUDFOUNDRY_PROPERTIES + ".services";
 
+	public static final String HEALTHCHECK_PROPERTY_KEY = CLOUDFOUNDRY_PROPERTIES + ".health-check";
```

Comment:
The usage of the key `HEALTHCHECK_PROPERTY_KEY` doesn't include `stream` or `task` and being referred as `deployment` property from Deployment Request. Does this mean, when using as a stream deployment property it doesn't require `stream` or `task` in the prefix as in the App Deployer properties that are set via CF server startup?

I see all the other properties are handled the same way.